src/main/resources/static/css/side-navbar.css

```
@@ -43,4 +43,5 @@ li:hover{
     font-weight: 500;
     cursor: pointer;
 }
-{}
\ No newline at end of file
+{}
```

Stylelint (17.6.0)

[error] Unexpected empty block (block-no-empty)

(block-no-empty)